src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -159,7 +159,7 @@ def load_arguments(self, _):
         c.argument('aad_client_app_id')
         c.argument('aad_server_app_id')
         c.argument('aad_server_app_secret')
-        c.argument('aad_tenant_id')
+        c.argument('aad_tenant_id', deprecate_info=c.deprecate(hide='2.0.45', expired='2.0.46'))
```

Comment:
//CC @tjprescott for double checking the usage. I have never used this flag since Travis dsigned it.

Reply:
My recommendation would be for the expiration to be `2.1.0`. That will coincide with the next "breaking" change of core.